src/test/java/org/takes/rs/RsPrettyJsonTest.java

```diff
@@ -91,13 +93,21 @@ public void reportsCorrectContentLength() throws Exception {
     }
 
     /**
-     * RsPrettyJSON can conform to equals and hash code contract.
+     * RsPrettyJSON can conform to equals.
      * @throws Exception If some problem inside
      */
     @Test
     public void conformsToEqualsAndHashCode() throws Exception {
-        EqualsVerifier.forClass(RsPrettyJson.class)
-            .withRedefinedSuperclass()
-            .verify();
+        new Assertion<>(
+            "Must evaluate true equality",
+            new RsPrettyJson(
+                new RsWithBody("{\"test\": {\"test\": \"test\" }}")
```

Comment:
@fanifieiev please, use different string values to each, instead of `"text"` for all.